packages/manager/apps/web/client/app/hosting/multisite/translations/Messages_fr_FR.json

```diff
@@ -8,6 +8,14 @@
   "hosting_multisite_statistics_period_week": "Semaine",
   "hosting_multisite_statistics_period_month": "Mois",
   "hosting_multisite_statistics_period_year": "Année",
+  "hosting_multisite_dialog_title": "Verification des serveurs DNS",
+  "hosting_multisite_dialog_close": "Fermer",
+  "hosting_multisite_dialog_modify_a_record": "Modifiez le champ de type A avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} geré sur les serveurs DNS {{dns}}",
+  "hosting_multisite_dialog_modify_a_record_same_nic": "Modifier le champ de type A",
```

Comment:
```suggestion
  "hosting_multisite_dialog_modify_a_record_same_nic": "Modifiez le champ de type A",
```

Reply:
I am not sure for this one :) 

![image](https://github.com/ovh/manager/assets/8767496/08254cd8-e5b8-4894-830d-eacadea3f036)

Reply:
Here is the rule:

In the manager, for CTA we always use 'infinitif' (to prompt the user to do the action, but in a polite way), and for description it can be 'imperatif'